docs/data-migration.md

```diff
@@ -0,0 +1,105 @@
+# Data migration
+
+This document describes the process of migrating `lnd`'s database state from one
+type of database backend (for example the `bbolt` based database files `*.db`
+such as the `channel.db` or `wallet.db` files) to another (for example the new
+remote database backends such as `etcd` or `postgres` introduced in
+`lnd v0.14.0-beta`).
+
+## Prepare the destination database
+
+### Using etcd as the destination remote database
+
+When using `etcd` as the main remote database, some specific environment
+variables need to be set to ensure smooth operations both during the data
+migration and also later for running `lnd` in production.
+
+Make sure you are running **at least `etcd v3.5.0` or later** with the following
+environment variables (or their command line flag counterparts) set:
+
+```shell
+# Allow lnd to batch up to 16k operations into a single transaction with a max
+# total TX size of 104MB.
+ETCD_MAX_TXN_OPS=16384
+ETCD_MAX_REQUEST_BYTES=104857600
+
+# Keep 10k revisions for raft consensus in clustered mode.
+ETCD_AUTO_COMPACTION_RETENTION=10000
+ETCD_AUTO_COMPACTION_MODE=revision
+
+# Allow the total database size to be up to 15GB. Adjust this to your needs!
+ETCD_QUOTA_BACKEND_BYTES=16106127360
+```
+
+Make sure you set the `ETCD_QUOTA_BACKEND_BYTES` to something that is
+sufficiently larger than your current size of the `channel.db` file!
+
+### Using postgres as the destination remote database
+
+Prepare a user and database as described in the [Postgres](postgres.md)
```

Comment:
shouldn't the link be: https://github.com/lightningnetwork/lnd/blob/master/docs/postgres.md?